docs/source/common_issues.rst

```diff
@@ -533,6 +533,7 @@ Here's the above example modified to use ``MYPY``:
    def listify(arg: 'bar.BarClass') -> 'List[bar.BarClass]':
        return [arg]
 
+.. _not-generic-runtime:
```

Comment:
Reading now through this section I think it makes sense to add couple sentences at the end about using `from __future__ import annotations` as a (better) alternative to string quotes on Python 3.7+.